### ISSUE TITLE
Repeated crash on startup

### ISSUE THREAD

**Issue by bztn** [OpenTTD/OpenTTD#9060]
[crash.log](https://github.com/OpenTTD/OpenTTD/files/6333860/crash.log)

## Version of OpenTTD
1.11.1 (steam)
## Expected result
The program starting
## Actual result
OpenTTD does not start (multiple tries)
## Steps to reproduce
I start the game via steam, it opens briefly and then crashes showing the crash dialog box.

```
*** OpenTTD Crash Report ***

Crash at: Mon Apr 19 07:00:14 2021
In game date: 2095-05-19 (20)

Crash reason:
 Exception: C0000005
 Location:  000000013F4C4CAD
 Message:   <none>

OpenTTD version:
 Version:    1.11.1 (0)
 NewGRF ver: 1b086d64
 Bits:       64
 Endian:     little
 Dedicated:  no
 Build date: Apr 18 2021 21:16:51

Registers:
 RAX: 00000000FF000000 RBX: 0000000000147580 RCX: 0000000000147580 RDX: 0000000000000000
 RSI: 0000000000000346 RDI: 0000000000000000 RBP: 0000000000000640 RSP: 00000000002DF590
 R8:  0000000000000002 R9:  0000000000000008 R10: 0000000000000002 R11: 0000000000000008
 R12: 0000000000147580 R13: 0000000000000640 R14: 0000000000000020 R15: 000000000051D600
 RIP: 000000013F4C4CAD EFLAGS: 00010202

 Bytes at instruction pointer:
 F3 AB B9 EC 88 00 00 48 8B 5C 24 30 48 83 C4 20 5F 48 FF 25 DB 64 E4 01

Stack trace:
 00000346 00000000 00147580 00000000 00000640 00000000 033B2A90 00000000
 033B2A90 00000000 3F4C17FB 00000001 033AE130 00000000 00000346 00000000
 033AE130 00000000 07956650 00000000 3F8BFE10 00000001 021F08E0 00000000
 033F0A90 00000000 033B2A90 00000000 3F8BFE50 00000001 07956650 00000000
 3F8BFE50 00000001 07956650 00000000 00000346 00000000 00000001 00000000
 00000640 00000000 3F4C9F61 00000001 033B2A90 00000000 021F08E0 00000000
 3F8BFE10 00000001 FF000000 00000000 3F8A9028 00000001 3F4C8BCD 00000001
 FFFFFFFF FFFFFFFF 3F8A8210 00000001 033B2A90 00000000 3F4CB5BD 00000001
 00000001 00000000 3F4C9EA9 00000001 00000003 00000000 00000000 00000000
 3F8BFE10 00000001 3F8A9028 00000001 3F8A9028 00000001 3F4CC0AC 00000001
 FFFFFFFF FFFFFFFF 3F4CC648 00000001 FFFFFFFF FFFFFFFF 3F8A8210 00000001
 00000000 00000000 70623034 6E612D70 00006D69 00000000 0000000A 00000000
 0000000F 00000000 0A77946C 0000D19D FFFFFFFF FFFFFFFF 3F8BFE10 00000001
 00000000 00000000 033B2AD0 00000000 033B2A90 00000000 3F4CC3BF 00000001
 B474A03F 00000106 002DF8B0 00000000 033B2B20 00000000 002DF8B0 00000000
 00000000 00000000 3F4C8B9F 00000001 033B2B20 00000000 033B2AD0 00000000
 033B2A90 00000000 00000000 00000000 033F0A00 00000000 3F609CD9 00000001
 033F0A00 00000000 033F0A00 00000000 00000000 00000000 00000000 00000000
 000003F0 00000000 0A7795FC 0000D19D 002DF760 00000000 00000001 00000000
 0000000A 00000000 002DF740 00000000 00000000 00000001 00000000 00000000
 00000000 00000000 00000000 00000000 00000000 000007FE 002DFCA8 00000000
 3F87F350 00000001 00000000 00000000 00000000 00000000 002DFCA0 00000000
 00000000 00000000 3F435D74 00000001 0000027F 00000000 00000000 00000000
 00000000 00000000 00001FA0 0000FFFF 00000000 00000000 00000000 00000000

Operating system:
 Name:     Windows
 Release:  6.1.7601 (Service Pack 1)
 Compiler: MSVC 1928

Configuration:
 Blitter:      40bpp-anim
 Graphics set: OpenGFX (7761)
 Language:     C:\Program Files (x86)\Steam\steamapps\common\OpenTTD\lang\english.lng
 Music driver: dmusic
 Music set:    OpenMSX (7761)
 Network:      no
 Sound driver: win32
 Sound set:    OpenSFX (7758)
 Video driver: win32-opengl

Fonts:
 Small:  sprite
 Medium: sprite
 Large:  sprite
 Mono:   sprite

AI Configuration (local: 0) (current: 0):
  0: Human
  1: Human
  2: Human
  3: Human
  4: Human
  5: Human

Libraries:
 LZMA:       5.2.5
 LZO:        2.10
 PNG:        1.6.37
 Zlib:       1.2.11

Module information:
 C:\Program Files (x86)\Steam\steamapps\common\OpenTTD\openttd.exe handle: 000000013F240000 size: 8553984 crc: 9A796EBF date: 2021-04-19 06:56:24
 C:\Windows\SYSTEM32\ntdll.dll handle: 0000000076F30000 size: 1732864 crc: 0341F36C date: 2016-03-17 23:01:02
 C:\Windows\system32\kernel32.dll handle: 0000000076E10000 size: 1163264 crc: 3193169B date: 2016-03-17 22:53:15
 C:\Windows\system32\KERNELBASE.dll handle: 000007FEFCD00000 size: 419840 crc: 95E3C9E1 date: 2016-03-17 22:53:15
 C:\Windows\system32\OPENGL32.dll handle: 000007FEF0510000 size: 1039872 crc: 821E408D date: 2009-07-14 01:41:53
 C:\Windows\system32\msvcrt.dll handle: 000007FEFED40000 size: 634880 crc: 9D1B448A date: 2011-12-16 08:46:06
 C:\Windows\system32\ADVAPI32.dll handle: 000007FEFE830000 size: 880640 crc: 26597F7A date: 2016-03-17 22:50:09
 C:\Windows\SYSTEM32\sechost.dll handle: 000007FEFD410000 size: 113664 crc: 98A6ADF4 date: 2015-05-25 18:19:09
 C:\Windows\system32\RPCRT4.dll handle: 000007FEFD4D0000 size: 1212928 crc: BE1702EE date: 2016-03-17 22:57:21
 C:\Windows\system32\GDI32.dll handle: 000007FEFD600000 size: 405504 crc: 5576A1EE date: 2015-12-08 19:07:28
 C:\Windows\system32\USER32.dll handle: 0000000076D10000 size: 1008640 crc: 390AABAD date: 2015-11-10 18:55:26
 C:\Windows\system32\LPK.dll handle: 000007FEFF230000 size: 41472 crc: 865F1B1B date: 2016-02-05 18:54:45
 C:\Windows\system32\USP10.dll handle: 000007FEFE480000 size: 802304 crc: 3DC2D6D4 date: 2015-11-03 19:04:51
 C:\Windows\system32\GLU32.dll handle: 000007FEF11D0000 size: 165376 crc: 3AEC0CD2 date: 2009-07-14 01:40:56
 C:\Windows\system32\DDRAW.dll handle: 000007FEF0410000 size: 569344 crc: 536DA9DD date: 2009-07-14 01:40:28
 C:\Windows\system32\DCIMAN32.dll handle: 000007FEF8C90000 size: 14336 crc: 1F4388EF date: 2016-02-05 18:53:52
 C:\Windows\system32\SETUPAPI.dll handle: 000007FEFF000000 size: 1900544 crc: A364BD6A date: 2010-11-21 03:24:28
 C:\Windows\system32\CFGMGR32.dll handle: 000007FEFCFD0000 size: 207872 crc: 1984C339 date: 2010-11-21 03:24:00
 C:\Windows\system32\OLEAUT32.dll handle: 000007FEFE990000 size: 862208 crc: 6A660D7C date: 2016-02-03 18:58:21
 C:\Windows\system32\ole32.dll handle: 000007FEFE550000 size: 2084864 crc: A92D057C date: 2016-03-17 22:56:24
 C:\Windows\system32\DEVOBJ.dll handle: 000007FEFD050000 size: 93184 crc: A7541AE2 date: 2009-07-14 01:40:28
 C:\Windows\system32\dwmapi.dll handle: 000007FEFAAB0000 size: 82944 crc: F979E48B date: 2015-07-09 17:58:26
 C:\Windows\system32\WS2_32.dll handle: 000007FEFF1E0000 size: 297984 crc: 0F68689B date: 2010-11-21 03:24:28
 C:\Windows\system32\NSI.dll handle: 000007FEFEE60000 size: 13824 crc: 53B46F6C date: 2009-07-14 01:41:53
 C:\Windows\system32\WINMM.dll handle: 000007FEF9010000 size: 217600 crc: 7BC1529F date: 2009-07-14 01:41:56
 C:\Windows\system32\IMM32.dll handle: 000007FEFE760000 size: 167424 crc: 41B2805A date: 2009-07-14 01:41:09
 C:\Windows\system32\MSCTF.dll handle: 000007FEFD300000 size: 1067520 crc: 362B3A40 date: 2015-01-17 02:48:38
 C:\Windows\system32\SHELL32.dll handle: 000007FEFD670000 size: 14179840 crc: E9F874F2 date: 2016-01-22 06:19:58
 C:\Windows\system32\SHLWAPI.dll handle: 000007FEFEDE0000 size: 448512 crc: 086DF743 date: 2010-11-21 03:24:22
 C:\Program Files (x86)\Steam\gameoverlayrenderer64.dll handle: 000007FEFA3F0000 size: 1809640 crc: 8A9EFDDF date: 2021-04-12 20:03:30
 C:\Windows\system32\PSAPI.DLL handle: 0000000077100000 size: 9216 crc: 84B5E9BA date: 2009-07-14 01:41:53
 C:\Windows\system32\api-ms-win-core-synch-l1-2-0.DLL handle: 000007FEF9740000 size: 12128 crc: 36734AD6 date: 2015-11-19 14:07:55
 C:\Windows\system32\uxtheme.dll handle: 000007FEFAE30000 size: 332288 crc: B0A13DF9 date: 2009-07-14 01:41:56
 C:\Windows\system32\ig7icd64.dll handle: 000007FEB7970000 size: 11652096 crc: 66797C56 date: 2016-03-01 23:53:32
 C:\Windows\system32\CRYPTBASE.dll handle: 000007FEFCB10000 size: 43520 crc: 7CC21F4B date: 2016-03-17 22:50:56
 C:\Program Files\ThinkPad\Bluetooth Software\btmmhook.dll handle: 000007FEC6660000 size: 241880 crc: 922A3AE3 date: 2013-05-14 11:28:56
 C:\Windows\system32\CLBCatQ.DLL handle: 000007FEFE790000 size: 607744 crc: 939E35C9 date: 2009-07-14 01:40:15
 C:\Windows\System32\MMDevApi.dll handle: 000007FEFB0F0000 size: 284160 crc: CDF2EF81 date: 2009-07-14 01:41:26
 C:\Windows\System32\PROPSYS.dll handle: 000007FEFAFB0000 size: 1212416 crc: B83FA3BE date: 2010-11-21 03:23:55
 C:\Windows\system32\AUDIOSES.DLL handle: 000007FEF4BB0000 size: 296448 crc: E010A4A9 date: 2015-02-03 03:30:55
 C:\Windows\system32\mswsock.dll handle: 000007FEFC450000 size: 327168 crc: 2C87855E date: 2016-04-22 14:14:06
 C:\Windows\System32\wshtcpip.dll handle: 000007FEFBAC0000 size: 13312 crc: 5D1E2C55 date: 2009-07-14 01:41:58
 C:\Windows\system32\wdmaud.drv handle: 000007FEF2D30000 size: 217088 crc: 12829973 date: 2010-11-21 03:24:32
 C:\Windows\system32\ksuser.dll handle: 0000000073470000 size: 5120 crc: 781141DD date: 2015-12-08 19:07:31
 C:\Windows\system32\AVRT.dll handle: 000007FEFB0E0000 size: 18432 crc: 5CE53CB7 date: 2009-07-14 01:40:05
 C:\Windows\system32\msacm32.drv handle: 000007FEF4B00000 size: 25600 crc: 2B579A05 date: 2009-07-14 01:38:50
 C:\Windows\system32\MSACM32.dll handle: 000007FEF4890000 size: 83456 crc: 3D67538B date: 2009-07-14 01:41:27
 C:\Windows\system32\midimap.dll handle: 000007FEF4880000 size: 20480 crc: CB902A13 date: 2009-07-14 01:41:23
 C:\Windows\System32\dmusic.dll handle: 000007FEFA7A0000 size: 125952 crc: 11947851 date: 2009-07-14 01:40:31
 C:\Windows\System32\DSOUND.dll handle: 000007FEF4FD0000 size: 540672 crc: B6225596 date: 2009-07-14 01:40:33
 C:\Windows\System32\POWRPROF.dll handle: 000007FEFB140000 size: 167424 crc: F6D04424 date: 2009-07-14 01:41:53
 C:\Windows\system32\WINTRUST.dll handle: 000007FEFCE10000 size: 229376 crc: F491DD77 date: 2015-04-27 19:23:45
 C:\Windows\system32\CRYPT32.dll handle: 000007FEFCE60000 size: 1480192 crc: 749882D6 date: 2015-04-27 19:23:13
 C:\Windows\system32\MSASN1.dll handle: 000007FEFCCC0000 size: 46592 crc: 50F9CBBE date: 2010-11-21 03:24:22
 C:\Windows\System32\dmsynth.dll handle: 000007FEFA770000 size: 119296 crc: 05A882A4 date: 2009-07-14 01:40:31

---- gamelog start ----
Tick 0: new game started
Revision text changed to 1.10.3, savegame version 218, not modified, _openttd_newgrf_version = 0x1a086d64
New game mode: 1 landscape: 2
Tick 1280: game loaded
New game mode: 2 landscape: 2
Tick 1280: game loaded
New game mode: 1 landscape: 2
Tick 39224: game loaded
New game mode: 2 landscape: 2
Tick 39224: game loaded
New game mode: 1 landscape: 2
Tick 60493: settings changed
Setting changed: vehicle.never_expire_vehicles : 0 -> 1
Tick 52147: game loaded
New game mode: 2 landscape: 2
Tick 52147: game loaded
New game mode: 1 landscape: 2
Tick 39493: game loaded
New game mode: 2 landscape: 2
Tick 39493: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
New game mode: 2 landscape: 2
Tick 29222: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
Revision text changed to 1.11.1, savegame version 290, not modified, _openttd_newgrf_version = 0x1b086d64
New game mode: 0 landscape: 2
---- gamelog end ----

Recent news messages:

*** End of OpenTTD Crash Report ***

Decoded stack trace:
[00] openttd              0x000000013F4C4CAD
[01] openttd              0x000000013F4C17FB
[02] openttd              0x000000013F4C9F61
[03] openttd              0x000000013F4C8BCD
[04] openttd              0x000000013F4C9EA9
[05] openttd              0x000000013F4CC0AC
[06] openttd              0x000000013F4CC3BF
[07] openttd              0x000000013F4C8B9F
[08] openttd              0x000000013F609CD9
[09] openttd              0x000000013F440CF8
[10] openttd              0x000000013F7A7E92
[11] kernel32             0x0000000076E259BD BaseThreadInitThunk + 13
[12] ntdll                0x0000000076F5A2E1 RtlUserThreadStart + 33

*** End of additional info ***
```

**Comment by LordAro** [comment]
Can you also upload crash.dmp & other crash.* files? Should be in the same place as crash.log

**Comment by bztn** [comment]
[crash.zip](https://github.com/OpenTTD/OpenTTD/files/6333910/crash.zip)

**Comment by bztn** [comment]
I note similarities with #8967 , this is also on a laptop. It was working fine, presumably until 1.11.1 yesterday.

**Comment by LordAro** [comment]
Mm, only 1.11.1 contains fixes that are supposed to workaround such crashes. If you set `video_hw_accel = false` in openttd.cfg as mentioned in the issue, does that help at all?

**Comment by bztn** [comment]
Yes, it functions correctly with hardware acceleration off.

**Comment by Rocha-afk** [comment]
Eu inicio o jogo via Steam, ele abre brevemente e depois trava mostrando a caixa de diálogo de travamento.
Um tempo antes tava nas configurações e acabei clicando na aceleração de hardware, e depois disso n entra mais o jogo
O que eu posso fazer?
![image](https://user-images.githubusercontent.com/82814105/115231490-fef28f00-a0eb-11eb-8b3e-32b6557409e3.png)

g-translate:
>I start the game via Steam, it opens briefly and then crashes showing the crash dialog.
A while before I was in the settings and I ended up clicking on the hardware acceleration, and after that n the game enters
What can I do?

**Comment by LordAro** [comment]
@Rocha-afk Don't press that button! You can find your openttd.cfg file (likely C:\Users\\\<user>\Documents\OpenTTD\openttd.cfg ) and set `video_hw_accel = false`

Alternatively, ensure that you've updated 1.11.1, it may fix the issue

**Comment by glx22** [comment]
```
>	openttd.exe!ClearPixelBuffer<unsigned int>(unsigned __int64 len, unsigned int data) Ligne 879	C++
 	openttd.exe!OpenGLBackend::Resize(int w, int h, bool force) Ligne 923	C++
 	openttd.exe!VideoDriver_Win32OpenGL::AllocateBackingStore(int w, int h, bool force) Ligne 1426	C++
 	openttd.exe!VideoDriver_Win32Base::ClientSizeChanged(int w, int h, bool force) Ligne 879	C++
 	openttd.exe!VideoDriver_Win32OpenGL::AfterBlitterChange() Ligne 1400	C++
 	openttd.exe!VideoDriver::RealChangeBlitter(const char * repl_blitter) Ligne 109	C++
 	openttd.exe!VideoDriver::Tick() Ligne 164	C++
 	openttd.exe!VideoDriver_Win32Base::MainLoop() Ligne 870	C++
 	openttd.exe!openttd_main(int argc, char * * argv) Ligne 839	C++
 	openttd.exe!WinMain(HINSTANCE__ * hInstance, HINSTANCE__ * hPrevInstance, char * lpCmdLine, int nCmdShow) Ligne 437	C++
 	[Code externe]	
```

**Comment by Rocha-afk** [comment]
> @ Rocha-afk Não aperte esse botão! Você pode encontrar seu arquivo openttd.cfg (provavelmente C: \ Usuários \ <usuário> \ Documentos \ OpenTTD \ openttd.cfg) e definir`video_hw_accel = false`
> 
> Como alternativa, certifique-se de que você atualizou 1.11.1, isso pode corrigir o problema

Sim, estou nessa versão!
Mas devido eu ter clicado o jogo não entra mais, aparece a tela preta do início por uns 3 segundos, ai o jogo fecha e aparece a mensagem da foto, ja tentei via steam e fora da steam, mas n adianta. Aparentemente eu devo ser desativado ou ativado a aceleração de hardware. 

g-translate:
>Yes, I'm in that version!
But because I clicked the game no longer enters, the black screen of the beginning appears for about 3 seconds, then the game closes and the message of the photo appears, I've tried it via steam and out of steam, but it doesn't work. Apparently I must be disabled or enable hardware acceleration.
